include/seqan3/alphabet/concept_pre.hpp

```diff
@@ -260,6 +260,9 @@ struct pseudoknot_support{};
  * \attention Do not specialise this shortcut, instead specialise seqan3::pseudoknot_support.
  */
 template<typename alphabet_type>
+//!\cond
+    requires requires (alphabet_type alph) { pseudoknot_support<alphabet_type>::value; }
+//!\endcond
```

Comment:
`_v` helpers should not require concepts.

It should return `false` if it has no support.

Change the base class to:
```c++
template<typename alphabet_type>
struct pseudoknot_support :
    std::false_type
{};
```

Reply:
No, pseudoknot_support is a static class variable of type uint8_t. 
I implemented it in the same way as alphabet_size_v, which has a concept requirement.
Without the concept check, the rna_structure_concept would not evaluate to `false` on dssp9, but rather produce a compiler error when pseudoknot_support::value is not found.

Reply:
The name of `alphabet_size_v` shows that it gives a size back. `pseudoknot_support` does not hint in any way that this will return a number. It sound to me like a boolean and the correlating question: `Has this thing pseudoknot support?`. Is this for a biologist / bioinformatician clear that this returns a number and what this number represents?

This is only a question and out of the scope of this pull request. Thank you for clarifying!

> Without the concept check, the rna_structure_concept would not evaluate to false on dssp9, but rather produce a compiler error when pseudoknot_support::value is not found.

This in itself is clearly a bug on gcc's side and knowing that `alphabet_size_v` uses the same trick, I think Hannes already reported it.

Reply:
You are right and indeed formerly the variabe was boolean. But I need to query the allowed nestedness in structure I/O. I was already considering to name it `max_pseudoknot_depth`, but that sounds very technical. In your opinion, is that more intuitive?